### PR TITLE
Create chrome-55-deprecations.md

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 55,
+which is in beta as of October 21. This list is subject to change at any time.
+
+## Resources with non-script MIME types can no longer be executed
+
+Previous versions of Chrome allowed content to with several non-script [MIME
+types](https://developer.mozilla.org/en-
+US/docs/Web/HTTP/Basics_of_HTTP/MIME_types){: .external} to be executed as
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: don't line break URLs, it breaks markdown

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+script. In addition to the obvious security vulnerability, this problem also
+reduces the value of [content security policy](https://developer.mozilla.org/en-
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: URL line breaks

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+upload JavaScript embedded within a specially-crafted image file, and that
+JavaScript had the potential to be served from that origin and executed. As of
+Chrome 55 will no longer execute content loaded with the following MIME types:
+
+* `audio/*`
+* `image/*`
+* `video/*`
+* `text/csv` 
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/AHsFvhHzh1o/discussion){# .external} &#124;
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: what's the `&#124` for?

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+[Chromestatus Tracker](https://www.chromestatus.com/features/6031053726679040){# .external} &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=433049){# .external}
+
+## Remove SVGSVGElement.viewPort
+
+The implementation of `SVGSVGElement.viewPort` [has not worked in
+Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=395838){#
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: should be {: .external } not {# .external }

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+changes to specifications to bring alignment and consistency with other
+browsers, or they are early experiments that never came to fruition in other
+browsers and thus can increase the burden of support for web developers.
+
+Some of these changes will have an effect on a very small number of sites. To
+mitigate issues ahead of time, we try to give developers advanced notice so that
+if needed, they can make the required changes to keep their sites running.
+
+Chrome currently has a [process for deprecations and removals of
+API's](http://www.chromium.org/blink#TOC-Launch-Process:-Deprecation){#
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: don't split `){:` on to multiple lines, it breaks markdown

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+* Announce on the
+  [blink-dev](https://groups.google.com/a/chromium.org/forum/#!forum/blink-dev){# .external}
+  mailing list.
+* Set warnings and give time scales in the Chrome DevTools Console when usage
+  is detected on a page.
+* Wait, monitor, and then remove feature as usage drops.
+
+You can find a list of all deprecated features in [chromestatus.com using the
+deprecated filter](https://www.chromestatus.com/features#deprecated){#
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: {# vs {: and line breaks

File: src/content/en/updates/2016/10/chrome-55-deprecations.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+.external} and removed features by applying the [removed
+filter](https://www.chromestatus.com/features#removed){# .external}. We will
```
Comment (petele) [inline src/content/en/updates/2016/10/chrome-55-deprecations.md]: {: vs {#

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**